shell/browser/api/electron_api_app.cc

```diff
@@ -1071,16 +1071,15 @@ Browser::LoginItemSettings App::GetLoginItemSettings(
 }
 
 #if defined(USE_NSS_CERTS)
-void App::ImportCertificate(const base::DictionaryValue& options,
+void App::ImportCertificate(const base::Value& options,
```

Comment:
Can we make this [passed by value](https://source.chromium.org/chromium/chromium/src/+/master:base/values.h;l=72;drc=b01268ba46dd26506ea62c7ccfa63e00060bcd29) instead of by reference? I'm not sure it makes sense to `std::move` a const reference.